ignite/services/chain/init.go

```diff
@@ -126,7 +121,9 @@ func (c *Chain) InitAccounts(ctx context.Context, conf *chainconfig.Config) erro
 	c.ev.Send("🗂  Initialize accounts...")
 	c.ev.SendView(accounts)
 
-	_, err = c.IssueGentx(ctx, createValidatorFromConfig(conf))
+	if conf.Validators != nil {
+		_, err = c.IssueGentx(ctx, createValidatorFromConfig(conf))
+	}
```

Comment:
I think the check is not needed, having at least one validator is [required](https://github.com/ignite/cli/blob/develop/ignite/chainconfig/parser.go#L93) for the config to be considered valid.